web_src/js/features/comp/ComboMarkdownEditor.js

```diff
@@ -71,8 +71,19 @@ class ComboMarkdownEditor {
     this.textareaMarkdownToolbar = this.container.querySelector('markdown-toolbar');
     this.textareaMarkdownToolbar.setAttribute('for', this.textarea.id);
 
-    this.switchToEasyMDEButton = this.container.querySelector('.markdown-switch-easymde');
-    this.switchToEasyMDEButton?.addEventListener('click', async (e) => {
+    const monospaceButton = this.container.querySelector('.markdown-switch-monospace');
+    monospaceButton?.addEventListener('click', (e) => {
+      e.preventDefault();
+      const enabled = localStorage?.getItem('editor-monospace') !== 'true';
+      localStorage.setItem('editor-monospace', String(enabled));
+      this.textarea.classList[enabled ? 'add' : 'remove']('gt-mono');
```

Comment:
```suggestion
      this.textarea.classList.toggle('gt-mono', enabled);
```

Reply:
Seems equivalent. It seems so odd that this argument is called `force`, when it should be really called `present` or something.